conduct.md

```diff
@@ -11,8 +11,74 @@
 * Attendance
 * Deadlines
 * Communication
-* Participation
-* Decisions
+
+### Participation
+
+Detailed paragraph describing the category. (delete when added paragraph)
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+<<<<<<< HEAD
+    * Must contribute when designated to, in a timely manner
+    * Quick to respond in Slack to clarify confusing issues or assist other colleagues
+    * Everyone’s opinion must be considered
```

Comment:
This might need to be rephrased as `Consider everyone's opinion`. But again, is this necessary in participation, though? I'm not really sure.